calcom/cal.com
# Fixes bug when changing the location of a booking

## What does this PR do?

Fixes the following issues when changing the location of a booking:
- Wrong attendees receive location change email
- Calendar event with old location is not deleted

Loom Video: https://www.loom.com/share/d8c36c4edd5d4817b66d68c989a09af0

Fixes #3588 

## Type of change


## How should this be tested?

- Create a booking (with integrated calendar)
- Change the location
- Check if attendees received location change email 
- Check your calendar to see if the event has the new location

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
